<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtn_principal</class>
 <widget class="QMainWindow" name="vtn_principal">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ventana Principal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="txt_nombre">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_apellido">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_nombre">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_apellido">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Apellido:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_grabar">
    <property name="geometry">
     <rect>
      <x>194</x>
      <y>390</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Grabar</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_tipo_persona">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Docente</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Estudiante</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbl_tipo_persona">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_cedula">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cedula:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_email">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_cedula">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_email">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_buscar_cedula">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar por Cedula</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_estatura">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Estatura: (cm)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_estatura">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_estatura">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>E.D.Estatura</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_peso">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Peso: (libras)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_peso">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_peso">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>E.D.Peso</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="fecha_nac">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>310</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="date">
     <date>
      <year>1900</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_fecha_nac">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>310</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha_nac</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_fnacimiento">
    <property name="geometry">
     <rect>
      <x>214</x>
      <y>342</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular Edad </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="mnb_menu_principal">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>586</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="stb_estado"/>
 </widget>
 <tabstops>
  <tabstop>txt_apellido</tabstop>
  <tabstop>btn_grabar</tabstop>
  <tabstop>txt_nombre</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
